web/book/src/syntax/README.md

```diff
@@ -15,7 +15,6 @@ A summary of PRQL syntax:
 | `:`                              | [Named args & Parameters](../queries/functions.md)                      | `interp low:0 1600 sat_score`                           |
 | `[]`                             | [Lists](./lists.md)                                                     | `select [id, amount]`                                   |
 | <code>! && \|\| == +</code>, etc | [Expressions & Operators](./expressions-and-operators.md)               | <code>filter a == b + c \|\| d >= e</code>              |
-| `()`                             | [Precedence & Parentheses](./precedence-and-parentheses.md)             | `derive celsius = (fahrenheit - 32) / 1.8`              |
```

Comment:
The thing we really need to keep here is an entry in the table for parentheses!

Reply:
Good point. I'll add that later today. What about the other questions in that page?